src/.nuget/NuGet.Config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
   	<!--<add key="MyGet" value="https://www.myget.org/F/reactiveui/api/v3/index.json" />-->
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="Avalonia MyGet" value="https://www.myget.org/F/avalonia-ci/api/v2" />
```

Comment:
This won't be able to remain this way. You force any user to have to have a myget reference as well in their projects. Anything special you use in the beta not already in the nuget.org version? Eg if not you can just reference the latest nuget.org version with ="*" in the csproj, and you're not stopping people from using the myget anyway.

Reply:
When I reference Avalonia `0.6.1` (latest on nuget), I get into a version hell and get exceptions like this one when I try to build the project: `System.IO.FileNotFoundException: Could not load file or assembly 'System.Reactive.Interfaces`. Latest beta is the only version where it *just worked*. So we may continue working on it like this until Avalonia team releases a new version on NuGet or until we figure out how to fix that version hell...

Reply:
You add a reference to System.Reactive.Compatibility to the ReactiveUI.Avalonia.csproj file.